Formula/d/dug.rb

```diff
@@ -0,0 +1,23 @@
+class Dug < Formula
+  desc "Global DNS propagation checker that gives pretty output"
+  homepage "https://dug.unfrl.com"
+  url "https://github.com/unfrl/dug/archive/refs/tags/0.0.93.tar.gz"
```

Comment:
this tag did not get marked as latest release yet

Reply:
Thanks for raising this in the original repo!